### PR TITLE
hw 10 was added

### PR DESCRIPTION
**Pull request by serhii-k-n**

### CODE REVIEW

File: homework_10/define_calculator_first.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    result = multiply_numbers(num_1=first_number, num_2=second_number)
+    print(f'The multiplication of two numbers equals: {result}')
+elif operator == '/':
+    try:
+        result = divide_numbers(num_1=first_number, num_2=second_number)
+        print(f'The division of two numbers equals: {result}')
+    except ZeroDivisionError:
+        print('You cannot divide by 0')
+else:
+    print('You should choose one of these symbols (+, -, *, /). Try again')
```
Comment (MarinaZelenska) [inline homework_10/define_calculator_first.py:38]: Good) one more variant 
https://github.com/MarinaZelenska/test_repo_python/pull/7/files#diff-e38849d79e2ad0d5aea391b4c687352c5b6db1eea0f7c0e6db2286b3078b2314

File: homework_10/def_second.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+2. Напишіть функцію is_prime, яка приймає 1 аргумент - число від 2 до 1000, і повертає True, якщо це просте число, і False в іншому випадку.
+P.S просте число це те, що ділиться без залишку тільки на 1 і на самого себе
+
+Приклад простих чисел: 2, 3, 5, 7, 11, 13, 17, 19, 23, 29
+Приклад непростих чисел: 4, 6, 8, 9, 10, 12, 15, 21, 25
+'''
+
+
+def is_prime(number):
+    if (number < 2) or (number > 1000):
```
Comment (MarinaZelenska) [inline homework_10/def_second.py:11]: if number not in range(2, 1001):

File: homework_10/def_second.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+        return False
```
Comment (MarinaZelenska) [inline homework_10/def_second.py]: I suppose it should be better
    if number not in range(2, 1001):
        raise ValueError(f"{number} is not a valid number. Please, try any number in range from 2 to 1000.")

**Review by MarinaZelenska — APPROVED** [review]

File: homework_10/define_calculator_first.py (hunk 1 above)
Comment (serhii-k-n) [reply homework_10/define_calculator_first.py:38]: thanks

File: homework_10/def_second.py (hunk 3 above)
Comment (serhii-k-n) [reply homework_10/def_second.py]: done

**Pull request merged**